public/search.js

```diff
@@ -0,0 +1,204 @@
+        success: function(data, textStatus) {
+        console.log('got collections: ' + data);
+      },
+      error: function(jqXHR, textStatus, errorThrown) {
+        console.log('problem');
+      },
+      async: false,
+    });
+    docs = JSON.parse( docs.responseText );
+
+    //Checks and sets boundary points.
+    //Looks at the index of the first and last document (passed into the function)
+    //and disables or enables the next/previous buttons based on those indexes.
+    if( start <= 1 )
+    {
+      document.getElementById("prev").disabled = true;
+    }
+    else
+    {
+      document.getElementById("prev").disabled = false;
+    }
+
+    if( end >= docs.total)
+    {
+      document.getElementById("next").disabled = true;
+      end = docs.total;
+    }
+    else
+    {
+      document.getElementById("next").disabled = false;
+    }
+
+    var docLen = docs.total;
+    document.getElementById("numDocs").innerHTML = start.toString() + " to " + end.toString() + " of " + docLen;
+
+    //adds all document URI's into an array, in order to display the results alphabetically sorted
+    var uriArray = [];
+    for (var k = 0; k < docs.total; k++) {
+      uriArray[k] = docs['results'][k]['uri'];
+    }
+    uriArray.sort();
+
+    //nested for loop to loop through the different URI's in the sorted array of URI's, and add the URI to the bullet list.
+    //then under each uri the system start, system end, valid start, valid end times will be displayed
+    //NOTE: this outer loop will not display every URI in the array, it only displays a specified range 
+    for (var i=start; i < end; i++)
+    {
+       var uri = uriArray[i];
+       //getting a list of collections the document belongs to for each URI
+       var docColl = $.ajax(
+        {
+          url: "http://localhost:3000/v1/documents?uri="+uri+"&category=collections&format=json",
+            success: function(data, textStatus) {
+            console.log('got collections: ' + data);
+          },
+          error: function(jqXHR, textStatus, errorThrown) {
+            console.log('problem');
+          },
+          async: false,
+        });
+
+       docColl = JSON.parse(docColl.responseText);
+
+       var uriLogical;
+       var collArr = docColl.collections;
+       //gets the name of the logical for each physical document
+       for (var t = 0; t < collArr.length; t++) {
+         if (!collArr[t].includes( "latest" ) && !collArr[t].includes( selectedColl ) )
+         {
+           uriLogical = collArr[t];
+         }
+       }
+       bullet.append($("<hr id='break'>"));
+       bullet.append($("<em id= 'physicalDoc'>").text(uri + "   "));
+       bullet.append($("<a href = 'index' id='links'>").text("("+uriLogical+")"));
```

Comment:
href should be "/", or better: "/?collection=logical", where "logical" is the uriLogical.